syntax="proto2";

import "aap_protobuf/shared/MessageStatus.proto";

package aap_protobuf.service.media.sink.message;

message KeyBindingResponse {
    required int32 status = 1;
}